cmd/osbuild-composer/main.go

```diff
@@ -41,6 +49,7 @@ func main() {
 	logLevel, err := logrus.ParseLevel(config.LogLevel)
 
 	logrus.SetReportCaller(true)
+	logrus.AddHook(&common.ContextHook{})
```

Comment:
This hooks puts operation_id and external_id fields into the event.

Reply:
Since this isn't obvious, the comment would be more useful in the code instead of the in-line PR comments.